docs/lib/auth/fragments/flutter/signin_web_ui/20_platform_specific_setup.md

```diff
@@ -0,0 +1,45 @@
+## How It Works
+Signin with web UI requires the Amplify plugin to display the signin UI inside a webview. This is managed internally by amplify-android and amplify-ios. After the signin process is complete they will redirect back to your app.
+
+## Android Platform Setup
+Add the following activity to the `AndroidManifest.xml` file in your app's `Android` directory, replacing `myapp` with whatever value you used for your redirect URI prefix:
```

Comment:
Should this be lowercase for a directory? `android`?

```suggestion
Add the following activity to the `AndroidManifest.xml` file in your app's `android` directory. Replacing `myapp` with the "redirect signin URI" you provided to the CLI:
```

Reply:
Pushed a similar revision from my local.